infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/DirectKubernetesAPIAccessHelper.java

```diff
@@ -88,7 +88,9 @@ public static Response call(
     String mediaType = inputMediaType(headers);
 
     RequestBody requestBody =
-        body == null ? null : new InputStreamBasedRequestBody(body, mediaType);
+        body == null
+            ? null
+            : RequestBody.create(MediaType.parse(mediaType), IOUtils.toByteArray(body));
```

Comment:
Can you use guava(we already have a dependency on it) version?

```
ByteStreams.toByteArray
```
https://github.com/google/guava/blob/d3c306e9e4220d80f111348a92f5cb56d8ad965c/guava/src/com/google/common/io/ByteStreams.java#L219

BTW. What about adding some realistic limit ?https://github.com/google/guava/blob/d3c306e9e4220d80f111348a92f5cb56d8ad965c/guava/src/com/google/common/io/ByteStreams.java#L683. For example 1Mb?

Reply:
I did look at guava's `ByteStreams` but it is `@Beta` in the version we're using in che-server (or so my IDE told me). Cool it came out of beta in the meantime.